ports/fftw3/vcpkg.json

```
@@ -1,7 +1,7 @@
 {
   "name": "fftw3",
   "version": "3.3.10",
-  "port-version": 9,
+  "port-version": 12,
```

```suggestion
  "port-version": 10,
```
You do this only once in a PR.